cmd/kubectl-gadget/bcck8s.go

```diff
@@ -103,22 +96,6 @@ func init() {
 		utils.AddCommonFlags(command, &params)
 	}
 
-	// Add flags specific to some BCC gadgets
-	capabilitiesCmd.PersistentFlags().BoolVarP(
```

Comment:
I think we are now loosing this features.
We can make them back when we will have the CO-RE version of it.

Reply:
I think we can also make them work with this standard version but I prefer to make this in another PR to keep things moving now.